python/pyspark/ml/classification.py

```diff
@@ -857,6 +871,12 @@ def featureImportances(self):
         """
         return self._call_java("featureImportances")
 
+    @property
+    @since("2.0.0")
+    def trees(self):
```

Comment:
Oh, sorry for mistake comments. I commented at ```GBTClassifier.trees``` and meant to remove  ```GBTClassifier.trees``` at L747-L750.